### ISSUE TITLE
fine tuning ERROR

### ISSUE THREAD

**Issue by swc0717** [INFINIQ-AI1/RISF#1]
Do I need to change OUT_DIR and WEIGHTS in the configuration file when running the fine-tuning commands? If you do not change it, you will not find the file, and after changing it, you will report an error, and the error content is KeyError: 4

**Comment by INFINIQ-AI1** [comment]
Yes, you need to change OUT_DIR to save your model to any directory, and WEIGHTS to your pretrained weights directory.
If your device does not have pretrained weights, you can simply use WEIGHTS: "Detectron2://ImageNetPretrained/MSRA/R-101.pkl". Please upload the full code log for KeyError:4.

**Comment by swc0717** [comment]
Thank you very much for your reply. This is the detailed error information about the key-value pairs of the dictionary
![1](https://github.com/INFINIQ-AI1/RISF/assets/163231797/d583c645-92f6-4fe8-8a6b-fa1e722058fd)
![Uploading 1.jpg…]()

**Comment by INFINIQ-AI1** [comment]
Hi @swc0717 ! 😄
It looks like 2 is not included in self.class_mapper.
First, you can check whether "coco14_test_novel" is the test dataset in the configuration file.
If you use a custom dataset, you need to modify the class_mapper in "risf/evaluation/cmclip.py".
Second, please print the build process of class_mapper (42~45 of risf/evaluation/cmclip.py). You will find out what the problem is.

**Comment by qjh666888** [comment]
Excuse me, has it been resolved
